hibernate-core/src/main/java/org/hibernate/Hibernate.java

```diff
@@ -34,32 +34,31 @@
  * <p>
- * Operations like {@link #isInitialized(Object)} and {@link #initialize(Object)} are
- * of general purpose. But {@link #createDetachedProxy(SessionFactory, Class, Object)}
- * and {@link CollectionInterface#createDetachedProxy(SessionFactory, Class, Object)}
- * are intended for use by generic code that must materialize an "amputated" graph of
- * Hibernate entities. (For example, a library which deserializes entities from JSON.)
+ * Operations like {@link #isInitialized(Object)} and {@link #initialize(Object)} are of general purpose. But
+ * {@link #createDetachedProxy(SessionFactory, Class, Object)} and
+ * {@link CollectionInterface#createDetachedProxy(SessionFactory, Class, Object)} are intended for use by generic code
+ * that must materialize an "amputated" graph of Hibernate entities. (For example, a library which deserializes entities
+ * from JSON.)
  * <p>
- * Graphs of Hibernate entities obtained from a {@link Session} are usually in an
- * amputated form, with associations and collections replaced by proxies and lazy
- * collections. (That is, by instances of the internal types {@link HibernateProxy}
- * and {@link PersistentCollection}.) These objects are fully serializable using
- * Java serialization, but can cause discomfort when working with custom serialization
- * libraries. Therefore, this class defines operations that may be used to write code
- * that completely removes the amputated leaves of the graph (the proxies) during
- * serialization, and rematerializes and reattaches them during deserialization. It's
- * possible, in principle, to use these operations, together with reflection, or with
- * the Hibernate metamodel, to write such generic code for any given serialization
- * library, but the details depend on what facilities the library itself offers for
- * the program to intervene in the process of serialization and of deserialization.
+ * Graphs of Hibernate entities obtained from a {@link Session} are usually in an amputated form, with associations and
+ * collections replaced by proxies and lazy collections. (That is, by instances of the internal types
+ * {@link HibernateProxy} and {@link PersistentCollection}.) These objects are fully serializable using Java
+ * serialization, but can cause discomfort when working with custom serialization libraries. Therefore, this class
+ * defines operations that may be used to write code that completely removes the amputated leaves of the graph (the
+ * proxies) during serialization, and rematerializes and reattaches them during deserialization. It's possible, in
+ * principle, to use these operations, together with reflection, or with the Hibernate metamodel, to write such generic
+ * code for any given serialization library, but the details depend on what facilities the library itself offers for the
+ * program to intervene in the process of serialization and of deserialization.
```

Comment:
Please don't reformat code that you don't need to change, especially code unrelated to your patch.

Can you revert all those unnecessary reformattings, new lines, etc.?

Reply:
Sure

Reply:
It is ok to make a new commit reverting changes? Or is prefered to make a new "clean" history and force push the new commits to my branch?

Reply:
You can make a new commit and squash everything together? `git rebase -i main` from the commandline, and follow the instructions in the editor that will open. I'm sure graphical interfaces to git have similar features. Otherwise just push a new commit and I'll squash everything for you.